checker/build.xml

```diff
@@ -256,6 +254,20 @@
         <delete file="${build}/.timestamp"/>
     </target>
 
+    <target name="-compile-java8" depends="-force-compile-java8"
+            description="Compiles java 8 sources if JAVA_8_HOME is set or javac is already java8">
+        <antcall target="compile-java8-sources">
```

Comment:
Isn't this ant call redundant, as `-force-compile-java8" already compiled the same code?
For the javadoc targets, this looks nicely separated into targets that have different conditions.